attributeTcMsg.go

```diff
@@ -369,6 +369,15 @@ func extractXStats(data []byte, tc *XStats, kind string) error {
 		err := unmarshalFqCodelXStats(data, info)
 		multiError = concatError(multiError, err)
 		tc.FqCodel = info
+	case "fq":
+		info := &FqQdStats{}
+		// Pad out data to size of our FqQdStats struct to handle
+		// unmarshalling data from older kernel versions with smaller structs
+		qd := make([]byte, FqQdStatsSize)
```

Comment:
Can we just drop `FqQdStatsSize` and replace it with `binary.Size()`?
```suggestion
		qd := make([]byte, binary.Size(info))
```

Likely have to add `encoding/binary` as import.